src/Locale/de/cake.po

```diff
@@ -230,11 +230,11 @@ msgstr "Der angegebene Wert ist ungültig"
 
 #: View/Helper/FormHelper.php:963
 msgid "New %s"
-msgstr ""
+msgstr "%s hinzufügen"
 
 #: View/Helper/FormHelper.php:966
 msgid "Edit %s"
-msgstr ""
+msgstr "%s bearbeiten"
```

Comment:
But does that still work in 3.x? The default is that {syntax} - all the code should also use that already.
Just wondering why those files still contain the old placeholders here.